src/app/core/record-works/record-works.service.ts

```diff
@@ -182,13 +182,15 @@ export class RecordWorksService {
       )
   }
 
-  save(work: Work) {
+  save(work: Work, bibtex?: boolean) {
```

Comment:
```suggestion
  save(work: Work, {forceReload: boolean} =  {forceReload:false}  {
```

Makes this more readable but I don't think this is a blocker for the merge